posthog/urls.py

```diff
@@ -204,9 +204,10 @@ def opt_slash_path(route: str, view: Callable, name: Optional[str] = None) -> st
 
 urlpatterns = [
     # internals
-    opt_slash_path("_health", health),
-    opt_slash_path("_stats", stats),
+    # opt_slash_path("_health", health),
+    # opt_slash_path("_stats", stats),
```

Comment:
This page is actually used in WorkerStats.js. However, I actually think we can get rid of that entirely as it doesn't show useful information and tends to be confusing to people. Do you want to take that on? Should be easy

Reply:
I have made the changes as you wanted I hope thats what you were looking for incase not let me know ill update the pr 
thanks @timgl